src/cellars/cellar_v2.rs

```diff
@@ -113,6 +112,62 @@ pub fn get_call(function: StrategyFunction, cellar_id: String) -> Result<CellarV
 
             Ok(CellarV2Calls::SetShareLockPeriod(call))
         }
+        Function::SetupAdaptor(params) => {
+            validate_new_adaptor(&cellar_id, &params.adaptor, &V2_0_PERMISSIONS)?;
+            log_cellar_call(CELLAR_NAME, &SetupAdaptorCall::function_name(), &cellar_id);
+
+            let call = SetupAdaptorCall {
+                adaptor: sp_call_parse_address(params.adaptor)?,
+            };
+
+            Ok(CellarV2Calls::SetupAdaptor(call))
+        }
+        Function::InitiateShutdown(_) => {
+            log_cellar_call(
+                CELLAR_NAME,
+                &InitiateShutdownCall::function_name(),
+                &cellar_id,
+            );
+            let call = InitiateShutdownCall {};
+            Ok(CellarV2Calls::InitiateShutdown(call))
+        }
+        Function::LiftShutdown(_) => {
+            log_cellar_call(CELLAR_NAME, &LiftShutdownCall::function_name(), &cellar_id);
+            let call = LiftShutdownCall {};
+            Ok(CellarV2Calls::LiftShutdown(call))
+        }
+        Function::SetPlatformFee(params) => {
+            log_cellar_call(
+                CELLAR_NAME,
+                &SetPlatformFeeCall::function_name(),
+                &cellar_id,
+            );
+            let call = SetPlatformFeeCall {
+                new_platform_fee: params.amount,
+            };
+            Ok(CellarV2Calls::SetPlatformFee(call))
+        }
+        Function::SetStrategistPlatformCut(params) => {
+            log_cellar_call(
+                CELLAR_NAME,
+                &SetStrategistPlatformCutCall::function_name(),
+                &cellar_id,
+            );
+            let call = SetStrategistPlatformCutCall { cut: params.amount };
+            Ok(CellarV2Calls::SetStrategistPlatformCut(call))
+        }
+        Function::SetRebalanceDeviation(params) => {
```

Comment:
`SetRebalanceDeviation` in Cellar v2.2 has some checks on the args, should we replicate them here?

Reply:
I thought I recalled something like that.